code/components/gta-net-five/src/MumbleVoice.cpp

```diff
@@ -498,6 +505,8 @@ static HookFunction initFunction([]()
 	OnKillNetworkDone.Connect([]()
 	{
 		g_mumbleClient->SetAudioDistance(0.0f);
+		g_mumble.overrideAddress.empty();
```

Comment:
You're.. checking if it's empty, and discarding the result of that check. :P

`.clear()` or the universal (which will also work if it's a `net::PeerAddress`) `var = {};` should be preferred.